setup.py

```diff
@@ -45,6 +45,7 @@ def version(path):
     "click>=8.0.0",
     "pydantic",
     "SQLAlchemy>=1.4",
+    "SQLAlchemy<2.0.0",  # https://sqlalche.me/e/20/zlpr
```

Comment:
We already have a line for SQLAlchemy above. you can set two bounds using `'A>=1,<2',` as mentioned in https://packaging.python.org/en/latest/discussions/install-requires-vs-requirements/#id5

Reply:
updated, thanks